### DISCUSSION TITLE
Mobs burn in daylight

### DISCUSSION THREAD

**Discussion by itsMarkiS** [PurpurMC/Purpur#614] [__Dismissed__] [answered]
would be cool if we could control which mobs should burn in daylight or even in nighttime
for example: 
• phantons not burning in daylight (already a feature)
• drowneds dont burn in daylight
• bat burns in daylight
• slime burns in night

**Comment by katekerllenevich** [comment, marked as answer]
There is already an API for this, so if anyone knows how to handle conflicts between those two, hmu.

**Reply by granny** [reply]
Since there’s already API available, i’d say this is plugin territory. Not a feature that can added to purpur's config

**Reply by itsMarkiS** [reply]
well.. i thought it would fit purpur's config due to the phantom burn, life, and rideable configs idk lol

**Reply by granny** [reply]
I’m not saying it wouldn’t, but since there’s API available it’s not easily feasible
